encoding/serialization.go

```diff
@@ -61,25 +78,87 @@ func (c *Frame) DeserializeGnark(data []byte) (*Frame, error) {
 	return &f, nil
 }
 
-func (f *Frame) Encode() ([]byte, error) {
-	var buf bytes.Buffer
-	enc := gob.NewEncoder(&buf)
-	err := enc.Encode(f)
+const SerializedProofLength = bn254.SizeOfG1AffineCompressed
```

Comment:
Seems like strange placement, to plop this const here amongst functions. Was this intentional?